a10_neutron_lbaas/neutron_ext/common/constants.py

```diff
@@ -14,14 +14,23 @@
 
 from neutron.plugins.common import constants as nconstants
 
+from a10_openstack_lib.resources import a10_certificate
 from a10_openstack_lib.resources import a10_device_instance
 
 A10_DEVICE_INSTANCE_EXT = a10_device_instance.EXTENSION
 A10_DEVICE_INSTANCE = a10_device_instance.SERVICE
 
+A10_CERTIFICATE_EXT = a10_certificate.EXTENSION
+A10_CERTIFICATE = a10_certificate.SERVICE
+A10_CERTIFICATE_BINDING = a10_certificate.CERTIFICATE_BINDING

```

Comment:
Pointer to constant - I didn't want "a10_certificate" and its permutations being splatted all over... one var = easy to change.
________________________________
From: Cedric Shock [notifications@github.com]
Sent: Tuesday, June 21, 2016 9:02 AM
To: a10networks/a10-neutron-lbaas
Cc: Michael Durrant; Author
Subject: Re: [a10networks/a10-neutron-lbaas] Support for SSL Cert binding in listeners (#354)


In a10_neutron_lbaas/neutron_ext/common/constants.py<https://github.com/a10networks/a10-neutron-lbaas/pull/354#discussion_r67898610>:

>  from a10_openstack_lib.resources import a10_device_instance
>
>  A10_DEVICE_INSTANCE_EXT = a10_device_instance.EXTENSION
>  A10_DEVICE_INSTANCE = a10_device_instance.SERVICE
>
> +A10_CERTIFICATE_EXT = a10_certificate.EXTENSION
> +A10_CERTIFICATE = a10_certificate.SERVICE
> +A10_CERTIFICATE_BINDING = a10_certificate.CERTIFICATE_BINDING


What's A10_CERTIFICATE_BINDING needed for?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/a10networks/a10-neutron-lbaas/pull/354/files/5dc9ce576a8e1982a906a41b5c1d0b028658194c#r67898610>, or mute the thread<https://github.com/notifications/unsubscribe/AJlobIyQXZ6a14Hp32zeS57LkjnRD1RBks5qOAsagaJpZM4I6aIY>.